src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -257,15 +257,6 @@
     <!-- Non-null type is required for this node kind -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
     <Field Name="Operand" Type="BoundExpression"/>
-    
-    <!-- 
-      Normal address-of operators can only operate on non-moveable variables. 
-      Fixed statement address-of operators (note: only applies to the top-level
-      expression in the initializer, not nested expressions) can only operate
-      on moveable variables.  This flag indicates which sort of address-of
-      operator we're dealing with.
-      -->
-    <Field Name="IsFixedStatementAddressOf" Type="bool"/>
```

Comment:
We do not need this any more. 
It was a bit of an abuse to use this node with slightly different meaning as an initializer in certain fixed statements.

Now we use BoundFixedLocalCollectionInitializer in those cases, so we no longer need to special case this node.